### PR TITLE
Create SpaceX Traveler

### PR DESCRIPTION
**Pull request by abdallahmalima**
### In this part  We:
- Add the basic structure  of the project
- Add Rockets Module
- Add Mission Module

### CODE REVIEW

File: src/components/MyProfile.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+      </div>
+      <div className="rocketsCont">
+        <h3 className="heading">My Rockets</h3>
+
+        {rockets.map((rocket) => (
+          <ReservedRocket key={rocket.id} rocket={rocket} />
+        ))}
+        { rockets.length === 0 && <p className="firstJoin">&emsp; No Reserved Rocket </p>}
+      </div>
+    </div>
```
Comment (franclobo) [inline src/components/MyProfile.js:24]: - [ ] Please 🙏🏽 use the underlined style in the MyProfile link like you did with the others to be similar to the original design.
- [ ] Kindly use two columns to list the missions and rockets for a better user experience.

| Original design | Your design |
|---|---|
| image | image |

**Review by franclobo — CHANGES_REQUESTED** [review]
Hi @abdallahmalima and @Mukumbuta, 

# REQUIRED CHANGES ♻️ 

Good job so far!
You did it well 👏🏽 there are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!

the ofice

## Highlights

- No linter errors ✔️ 
- Correct Git flow ✔️ 
- Descriptive PR ✔️ 
- Professional README file ✔️ 

## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: src/components/MyProfile.js (hunk 1 above)
Comment (peterrobert) [reply src/components/MyProfile.js]: This change has been done well

**Review by peterrobert — APPROVED** [review]
Hi @abdallahmalima, 


### Highlights!

[ ] Good feature branch name ✔️
[ ] Descriptive pull request ✔️


Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread or tag me @peterrobert if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**